pkgs/tools/networking/tuntox/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, stdenv
+, cscope
+, fetchFromGitHub
+, fetchpatch
+, gcc
+, git
+, libevent
+, libopus
+, libsodium
+, libtoxcore
+, libvpx
+, pkg-config
+, python3
+, python3Packages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tuntox";
+  version = "0.0.10";
+
+  src = fetchFromGitHub {
+    owner = "gjedeer";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "sha256-c/0OxUH8iw8nRuVg4Fszf6Z/JiEV+m0B2ofzy81uFu8=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ cscope gcc git libopus libtoxcore libsodium libevent libvpx python3 ];
```

Comment:
`cscope gcc git` should be in `nativeBuildInputs` instead, otherwise cross-compiliation will fail. Are you sure you need to explicitly include GCC?

Reply:
Addressed with c830bee.

Omitting `gcc` causes the following error:

```
$ nix-build $NIXPKGS -A tuntox
this derivation will be built:
  /nix/store/qynf5316b7z1xkbr6k4jawhm767rx5v7-tuntox-0.0.10.drv
building '/nix/store/qynf5316b7z1xkbr6k4jawhm767rx5v7-tuntox-0.0.10.drv'...
unpacking sources
unpacking source archive /nix/store/mgrj6laxx9wk0j8msfnd7bhschq84a0x-source
source root is source
patching sources
applying patch /nix/store/yp18wkbmndkc9wkwp0412bbrirkp9j86-a646402f42e120c7148d4de29dbdf5b09027a80a..365d2e5cbc0e3655fb64c204db0515f5f4cdf5a4.patch
patching file Makefile.mac
patching file mach.c
patching file mach.h
configuring
no configure script, doing nothing
building
Makefile.mac:19: warning: ignoring prerequisites on suffix rule definition
gcc -g  -I/nix/store/c14zp5411mam2hfs8cirwrixvdrlssh0-libtoxcore-0.2.13/include client.c -c -o client.o
make: gcc: No such file or directory
make: *** [Makefile.mac:19: client.o] Error 127
```

Reply:
We don't want to limit ourselves to GCC if we can avoid it, can `gcc` be replaced with `$CC` in `postPatch`?

Reply:
Of course, I wasn't thinking. Yes, `clang` works too. Adjusted with 5a0d4f26b98b52b95b3090a4fe1b4d001c3627e2.